src/ui/popup/main-page/app-switch.tsx

```diff
@@ -42,15 +42,13 @@ export default function AppSwitch(props: ViewProps) {
         }
     }
 
-    const descriptionText = isOn ?
-        'Extension is enabled' :
-        isOff ?
-            'Extension is disabled' :
-            isTimeAutomation ?
-                'Switches according to specified time' :
-                isLocationAutomation ?
-                    'Switched according to location' :
-                    'Switches according to system dark mode';
+    const descriptionText = (() => {
```

Comment:
`(() =>`
A new closure to get the variable can cause memory leaks as this is called a lot due it's the main page for older version of browsers(Bugs that are already fixed but compatability) so let's use the old `? : ? :`

Reply:
👍